docs/design/org-badges.html

```
@@ -0,0 +1,1384 @@
+<!--
+  Design exploration: organisation marks on programme cards.
+  Standalone page — open directly in a browser, or publish as an Artifact.
+  Fonts are inlined, so it works offline. No <html>/<head>/<body> wrapper:
+  the Artifact publisher supplies one.
+-->
+<title>Znaczki organizacji na kartach programu</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)